saltgui/static/scripts/Api.js

```diff
@@ -7,8 +7,8 @@ export class HTTPError extends Error {
 }
 
 export class API {
-  constructor(router, apiurl="") {
-    this.APIURL = apiurl;
+  constructor(router) {
+    this.APIURL = config.API_URL;
```

Comment:
can this assignment be removed?
and let function `apiRequest` directly use `config.API_URL`